lib/package.gi

```diff
@@ -2044,7 +2044,7 @@ InstallGlobalFunction( DeclareAutoreadableVariables,
 #F  ValidatePackageInfo( <info> )
 ##
 InstallGlobalFunction( ValidatePackageInfo, function( info )
-    local record, pkgdir, i, IsStringList, IsRecordList, IsProperBool,
+    local record, pkgdir, i, s, IsStringList, IsRecordList, IsProperBool, IsURL,
```

Comment:
Then it does't need to be declared at the outer level. It's implicitly declared by `s ->`